volunteer/index.php

```diff
@@ -105,7 +107,19 @@ function wdnInclude($path)
 		<!-- TemplateBeginEditable name="maincontentarea" -->
 		<div class="dcf-bleed">
 			<div class="dcf-wrapper dcf-pb-8">
-				<p>This page isn't quite ready. Check back later for updates!</p>
+			<p>Children living in poverty need your help. The American Rescue Plan of 2021 has increased the dollar amount of
+			 child tax credits and set the income limit to $0. There are 38,000 non-filers in Nebraska which include 9.600
```

Comment:
Comma instead of `.`?
```suggestion
			 child tax credits and set the income limit to $0. There are 38,000 non-filers in Nebraska which include 9,600
```